### PR TITLE
walking changes from terrain and stairs tests

### PR DESCRIPTION
**Pull request by kuindersma**

### CODE REVIEW

File: systems/robotInterfaces/@Biped/getPelvisMotionForWalking.m [hunk 1]
```diff
@@ -161,14 +161,14 @@
...
   elseif isLeftSupport && nextIsDoubleSupport 
-    if rsole_des_next(3) < lsole_des(3) - step_height_delta_threshold
+    if rsole_des_next(3) < lsole_des(3)
       % stepping down with right foot
       pelvis_reference_height(i+1) = rsole_des_next(3);
     else
       pelvis_reference_height(i+1) = lsole_des(3);
     end
   elseif isRightSupport && nextIsDoubleSupport 
-    if lsole_des_next(3) < rsole_des(3) - step_height_delta_threshold
```
Comment (RussTedrake) [inline systems/robotInterfaces/@Biped/getPelvisMotionForWalking.m]: is `step_height_delta_threshold` now unused?  if so, let's zap it.

**Comment by RussTedrake** [comment]
otherwise looks good to me.

**Comment by rdeits** [comment]
:

**Comment by rdeits** [comment]
:+1:

File: systems/robotInterfaces/@Biped/getPelvisMotionForWalking.m (hunk 1 above)
Comment (kuindersma) [inline systems/robotInterfaces/@Biped/getPelvisMotionForWalking.m]: we're still using it for stepping up

**Pull request merged**